nautobot/ipam/factory.py

```diff
@@ -238,9 +238,9 @@ def locations(self, create, extracted, **kwargs):
                         lambda: Location.objects.filter(location_type__content_types__in=[vlan_ct]), minimum=0
                     )
                 )
-                if self.vlan_group and self.vlan_group.location:
+                if self.vlan_group and self.vlan_group.location:  # pylint: disable=no-member
```

Comment:
The way factory classes work seems to frequently confuse `no-member` - it might be better to add a blanket `generated-members` rule in pyproject.toml for something like `.*Factory\..*`?